src/sentry/relay/config.py

```diff
@@ -397,7 +397,7 @@ def get_transaction_metrics_settings(
                     assert breakdown_config["type"] == "spanOperations"
 
                     for op_name in breakdown_config["matches"]:
-                        metrics.append(f"breakdown.{breakdown_name}.{op_name}")
+                        metrics.append(f"sentry.transactions.breakdowns.{breakdown_name}.{op_name}")
```

Comment:
Is it intentional to make it `breakdowns` rather than `breakdown`

Reply:
Yes, to be consistent with `measurements`.